resource-managers/mesos/src/main/scala/org/apache/spark/scheduler/cluster/mesos/MesosClusterScheduler.scala

```diff
@@ -427,6 +426,7 @@ private[spark] class MesosClusterScheduler(
   }
 
   private def getDriverUris(desc: MesosDriverDescription): List[CommandInfo.URI] = {
+    val useFetchCache = desc.conf.get(config.ENABLE_FETCHER_CACHE) || conf.get(config.ENABLE_FETCHER_CACHE)
```

Comment:
Could you fix the Scaly style error at this line? You can check with the following command in your PR directory.
```
$ dev/scalastyle
```